docs/source/guide/scripts.md

```diff
@@ -0,0 +1,208 @@
+
+!!! info Tip
+    Custom scripts are executed within an asynchronous function, so you can use `await` as necessary. 
+
+### Troubleshooting and debugging
+
+It is important to test and refine scripts using a test project first to avoid any disruptions on live projects. 
+
+Note the following:
+
+* Use the Console tab in your web browser’s developer tools to check for errors and verify the script is running. 
+* You can also check the Network tab (script information is returned with the `/project/:id` API call).
+* You can add [`debugger`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/debugger) to your script to have a convenient breakpoint to debug the script using your web browser’s developer tools. 
+
+### Security notes, constraints, and limitations
+
+Custom scripts are a powerful tool to help you fully customize your labeling workflow. In doing so you are running arbitrary JavaScript code on each annotator’s machine, which comes with certain risks. 
+
+Because of this, you must opt-in before enabling custom scripts for your organization, and we urge you to use this feature with caution.
+
+To enable custom scripts for your organization, you cannot have members that are in multiple organizations. This is enforced through application logic and is necessary for data security. The most common reason for this is when users have accounts in an expired free trial, but can also happen if you are using multiple organizations for project management or if you have an initial proof of concept or testing org. 
+
+## Label Studio Interface (LSI)
+
+The Label Studio Interface (LSI) is a helper object that is designed to be used with custom scripts. 
+
+LSI simplifies access to some data and can perform special actions that only make sense within the framework of custom scripts.
+
+### Instance methods
+
+##### `import(url, integrity)`
+
+Allows loading additional external scripts
+
+| Parameter  &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; | Type     | Description          |
+|--------------------------|--------------------|--------------------------|
+| `url` | string | Specifies the URL of an external script file.                                 |
+| `integrity`  | string | Allows a browser to check the fetched script to ensure that the code is never loaded if the source has been manipulated.  |
+
+The method is asynchronous, so you can wait for the script to load before performing the main actions. For example:
+
+```javascript
+await LSI.import('<https://cdn.plot.ly/plotly-2.26.0.min.js>', 'sha384-xuh4dD2xC9BZ4qOrUrLt8psbgevXF2v+K+FrXxV4MlJHnWKgnaKoh74vd/6Ik8uF');
+console.log("Plotly is ready");
+```
+
+##### `on(eventName, handler)`
+
+Subscription to listen to events related to the Label Studio Frontend. All handlers added within the framework of this method will be unsubscribed when switching to another annotation.
```

Comment:
> All handlers added within the framework of this method
Only the handlers attached/subscribed _by_ this method. Any handlers inside this method should be securet manually.